lib/main.dart

```diff
@@ -55,15 +55,15 @@ class _MyAppState extends State<MyApp> {
                       title: "",
                       bodyWidget: Center(
                         child: Text(
-                          "An App made with ❤️ in 🇮🇳",
+                          "An App made in 🇮🇳 with ❤️",
```

Comment:
This is an absurd change😅.

It should be `Made with ❤️ in 🇮🇳`

Reply:
Ohh sorry 😂 i just got confused between the two !!